src/components/taskDetails/index.tsx

```diff
@@ -158,6 +158,14 @@ const TaskDetails: FC<Props> = ({ taskID }) => {
         }
     }
 
+    function getTaskEstimates(value: number | undefined) {
+        if (value === undefined) {
+            return 'N/A';
+        } else {
+            return convertTimeStamp(value);
+        }
```

Comment:
Nit:
```suggestion
        if (value === undefined) {
            return 'N/A';
        } 
        return convertTimeStamp(value);
```

Else was not required.

Reply:
Fixed